src/test/setup_common.h

```diff
@@ -43,6 +45,16 @@ static inline void SeedInsecureRand(bool deterministic = false)
     g_insecure_rand_ctx = FastRandomContext(deterministic);
 }
 
+static inline void SeedInsecureRand(const uint256& seed)
+{
+    g_insecure_rand_ctx = FastRandomContext(seed);
+}
+
+static inline void SeedInsecureRandWithBoostRandomSeed()
+{
+    SeedInsecureRand(ArithToUint256(arith_uint256(boost::unit_test::runtime_config::get<unsigned>( boost::unit_test::runtime_config::btrt_random_seed ))));
```

Comment:
not sure how stable that boost interface is. They seem to change it every couple of versions

Reply:
Is there a better way to access this value? Maybe directly access the environment variable using std::getenv?